docs/source/reference/numpysupported.rst

```diff
@@ -210,13 +210,21 @@ The following methods of Numpy arrays are supported:
 * :meth:`~numpy.ndarray.repeat` (no axis argument)
 * :meth:`~numpy.ndarray.reshape` (only the 1-argument form)
 * :meth:`~numpy.ndarray.sort` (without arguments)
-* :meth:`~numpy.ndarray.sum` (with or without the ``axis`` argument. 
-  ``axis`` only supports ``integer`` values)
+* :meth:`~numpy.ndarray.sum` (with or without the ``axis`` and/or ``dtype`` arguments.)
 
+  * ``axis`` only supports ``integer`` values.
   * If the ``axis`` argument is a compile-time constant, all valid values are supported.
     An out-of-range value will result in a ``LoweringError`` at compile-time.
   * If the ``axis`` argument is not a compile-time constant, only values from 0 to 3 are supported.
     An out-of-range value will result in a runtime exception.
+  * All numeric ``dtypes`` are supported as ``dtype`` parameter. ``timedelta`` arrays can
```

Comment:
this section all needs line wrapping to 80 chars

Reply:
done

Reply:
err no :) I can go fix it though so don't worry!

Reply:
lol, I didn't know it had to be strictly 80. I thought it was like a pep8 thing, where there's a bit of flexibility.